src/components/StatusBadge/StatusBadge.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import StatusBadge from './StatusBadge';
+
+export default {
+  title: 'StatusBadge',
+  component: StatusBadge,
+  argTypes: {
+    status: { control: 'inline-radio', options: ['UNPAID', 'PARTIAL', 'PAID'] },
+  },
+} as ComponentMeta<typeof StatusBadge>;
+
+const Template: ComponentStory<typeof StatusBadge> = (args) => <StatusBadge {...args} />;
+
+export const UnpaidBadge = Template.bind({});
+UnpaidBadge.args = {
+  status: 'UNPAID',
```

Comment:
Might be better to have all 3 statuses in an object (emulating an enum) so this won't get repetitive and easier if there're new states.

I was thinking in something like 
```jsx
const statuses = {
  unpaid: "unpaid",
  partialPaid: "partial",
  paid: "paid"
};
```

And then you'd use it as:
```jsx
export const PartiallyPaidBadge = Template.bind({});
PartiallyPaidBadge.args = {
  status: statuses.partial,
};
```